shared/src/packages/orders/libs/types/order-find-by-request-dto.type.ts

```diff
@@ -0,0 +1,8 @@
+import { type OrderEntity } from './order-entity.type.js';
+
+type OrderFindByRequestDto = Pick<
```

Comment:
do we still use it somewhere in the code?

Reply:
removed